src/pdfpc.vala

```diff
@@ -122,8 +122,11 @@ namespace pdfpc {
                 ref Options.external_script,
                 "Enable execution of an external script", "file"},
             {"size", 'Z', 0, OptionArg.STRING,
-                ref Options.size,
-                "Size of the presentation window (implies \"-w\")", "W:H"},
+                ref Options.presentation_size,
+                "Size of the presentation window (implies \"-w both\")", "W:H"},
```

Comment:
Either `both` or `presentation`. But do keep the help message short, to ensure the `pdfpc -h` output lines are not wrapped in the standard 80-char-wide terminal. Hence, `-w` is probably enough.

Also, shouldn't it now be `WxH[+X+Y]`?

Reply:
> Either both or presentation.

But right now it does imply `both`, overriding any `-w` given on the command line.

> But do keep the help message short, to ensure the `pdfpc -h` output lines are not wrapped in the standard 80-char-wide terminal. Hence, `-w` is probably enough.

Well, it wasn't enough for me as I was a bit confused by the message but after seeing the screwed up linebreaks in my terminal I wanted to change it back and apparently plain forgot about it.

> Also, shouldn't it now be WxH[+X+Y]?

Sure, it should be mentioned. Do you mean that the `W:H` should not be mentioned anymore? (As long as the long form of `-A` is at least as long as `--pc-size`, describing the format as `(W:H|WxH[+X+Y])` will incur a line length of at least 81 for `--last-minutes` and 82 for `--size`. So, some sacrifice will have to be made, I guess.)

Reply:
> But right now it does imply `both`, overriding any `-w` given on the command line.

With two separate options, it no longer makes sense.

> Sure, it should be mentioned. Do you mean that the `W:H` should not be mentioned anymore?

I think so. Furthermore, if such a string is detected, I would issue a warning that this format is deprecated and will be removed soon.

Reply:
>> Sure, it should be mentioned. Do you mean that the W:H should not be mentioned anymore?
>
> I think so. Furthermore, if such a string is detected, I would issue a warning that this format is deprecated and will be removed soon.

:+1:

>> But right now it does imply both, overriding any -w given on the command line.
>
> With two separate options, it no longer makes sense.

Should the behaviour now be that `-w presenter -Z WxH` is equivalent to `-w both -Z WxH` and `-w none -Z WxH` to `-w presentation -Z WxH` where in the latter cases `-w` behaves exactly as advertised on its own? (Analogous for `-A`/`-z`/… of course and `-Z -A` implying `-w both` flat.)

Reply:
Thinking twice, maybe -[zZ] should *not* imply `-w`. Say, I want to start the presenter full screen but at some point make it windowed (pressing `w`) with a predefined size/position. Same about the presentation window (there is currently no action defined for toggling it full-screen/windowed, but can easily be added).

Reply:
So, essentially there are four options, right? Namely:
* `-[zZ]+` overrides `-w` (like now except different)
* `-[zZ]+` has no effect on windows that are full screen according to `-w` (which is still `-w none` unless explicitly specified otherwise)
* `-[zZ]+` does not interact with `-w` (equivalent to second option as long as there is no full screen toggle)
* `-[zZ]+` changes the default behaviour of `-w`

I'm partial to the last option. When giving `-Z` or `-z` and no other options it should probably have an effect and force the corresponding window to be windowed, but when giving `-zZ -w none` I'd just not be surprised to get two full screens and see the `-zZ` be “ignored”.

Will providing a full screen toggle creep into the scope of this PR? Because right now, I have no idea how to do that.

Reply:
I went ahead and implemented my favourite solution for now. I moved the documentation of this interaction into the man page section of `-w` because it felt more at home there. The short help does not mention this interaction at all in the current-as-of-writing-this-comment version of the PR.